libcloud/compute/drivers/kubevirt.py

```diff
@@ -167,10 +165,17 @@ def destroy_node(self, node):
         """
         namespace = node.extra['namespace']
         name = node.name
-        # stop the vmi first
+        # find and delete services for this VM only
+        services = self.ex_list_services(namespace=namespace, node_name=name)
+        for service in services:
+            service_type = service['spec']['type']
+            self.ex_create_service(node=node, ports=[],
```

Comment:
I think it would be better to have another ``ex_delete_service`` method which can call ``ex_create_service`` underneath with those arguments.

Also, from the method docstring it looks like you may need to specify ``override_existing_ports`` argument. Or is it not needed?

Reply:
Hey, 
An empty ports will result in the service being deleted. The `override_existing_ports` argument was added because there is no general standard in case there already are some ports forwarded and a user requests additional ports to be opened. Sometimes he might want to keep the previous, sometimes not. I thought it would be better to spare the user from manually deleting  in the second case (also I was not brilliant enough to have an `ex_delete_service` which makes a lot of sense).